net.sourceforge.jmol.yml

```
@@ -0,0 +1,47 @@
+app-id: net.sourceforge.jmol
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: jmol.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=home
+  - --device=dri
+  - --env=PATH=/app/jre/bin:/usr/bin
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: jmol
+    buildsystem: simple
+    build-commands:
+      - mv jmol/ /app/
+      - mkdir /app/bin
+      - mv jmol.sh /app/bin/jmol.sh
+      - chmod +x /app/bin/jmol.sh
+      - install -Dm644 files/net.sourceforge.jmol.desktop -t /app/share/applications/
+      - install -Dm644 files/net.sourceforge.jmol.png -t /app/share/icons/hicolor/128x128/apps/
+      - install -Dm644 files/net.sourceforge.jmol.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: archive
+        url: https://altushost-swe.dl.sourceforge.net/project/jmol/Jmol/Version%2016.1/Jmol%2016.1.27/Jmol-16.1.27-binary.zip
```

you should use the generic download URL  instead of a mirror.

```suggestion
        url: https://download.sourceforge.net/project/jmol/Jmol/Version%2016.1/Jmol%2016.1.27/Jmol-16.1.27-binary.zip
```